README.md

```diff
@@ -1,89 +1,162 @@
-important objectives, because it will improve the performance of other SYCL
-applications. Obviously, it makes sense SYCL-BLAS was the first step in this
-task.
-
-On GPUs, the data communication to/from the device and the grain of the kernels
-play an important rule on the performances of the developments. On one
-hand, to reduce the communication cost, the most of the data should be mapped
-on the device, even the scalars. On the other hand, growing the size of the
-kernels allows the CPU to complete other tasks while the GPU is computing or to
-enter an energy-efficient C-state, reducing the energy consumption.
-
-To enlarge the grain of the kernels is a complex task, in which many aspects
-should be considered as the dependency between kernels, the grid topology, the
-grid sizes, etc. This complexity justifies that, usually, the fused kernels are
-manually written. An alternative to simplify this task could be to build a
-expression tree on which all the single operation which are required to solve a
-problem appears. This structure could be analysed by the compiler to decide how
-to merge the different kernel and the best grid topology to execute the fused
-kernel.  The use of expression trees is one of most important features of
-SYCL-BLAS.
-
-## Basic Concepts
+libraries.
+
+The SYCL-BLAS project is used by the [SYCL-DNN project](https://github.com/codeplaysoftware/SYCL-DNN) 
+project as a supporting library to perform some operations. SYCL-DNN is a library 
+implementing neural network operations.
+
+SYCL-BLAS has been developed in collaboration with the 
+*High Performance Computing & Architectures (HPCA) group*
+from the Universitat Jaume I [UJI](http://www.hpca.uji.es/).
+
+The project is maintained by [Codeplay Software](https://developer.codeplay.com).
+
+## Project Requirements
+
+SYCL-BLAS is designed to work with any SYCL 1.2.1 implementation.
+* We do not use any OpenCL interoperability so the code is pure C++.
+* The project is developed using [ComputeCpp](https://developer.codeplay.com)
+ using Ubuntu 16.04 on Intel CPU and Intel GPU.
+* A BLAS library, such as OpenBLAS, is also required by the project to verify the test results
+
+
+## Supported Platforms
+
+The master branch of SYCL-BLAS is regularly tested with the "Supported" hardware 
+listed on [the ComputeCpp Supported Platforms page](https://developer.codeplay.com/products/computecpp/ce/guides/platform-support).
+
+SYCL-BLAS may also work on other hardware and platforms assuming they implement 
+ SPIR or SPIR-V support.
+
+## Getting Started with SYCL-BLAS
+
+### Pre-requisites
+
+* CMake (version 3.4.2 and above)
+* OpenCL 1.2-capable hardware and drivers with SPIR 1.2 or SPIR-V support
+* OpenCL ICD Loader
+* OpenCL headers
+* gcc (version 5.4 and above)
+* [ComputeCpp](https://developer.codeplay.com)
+
+### Building SYCL-BLAS
+
+First get OpenBLAS. Instructions on building and installing this are on 
```

Comment:
This is kind of a pre-requisite, not really part of building SYCL-BLAS.

Reply:
I wanted to explicitly describe how to do this part.